main.ts

```diff
@@ -39,7 +37,7 @@ await bundleTS();
 
 const sockets = new Set<WebSocket>();
 
-Deno.serve(async (req) => {
+Deno.serve({ port: 8000 }, async (req: Request) => {
```

Comment:
This only got changed because I was messing around for too long. I can revert it.